### PR TITLE
Post webpack code cleanup

### PR DESCRIPTION
**Pull request by bhollis**
* Removed remaining injector parameters (we do this automatically with ng-annotate now).
* Removed IIFE's wrapping all files (webpack does this for us).
* Remove global imports of underscore, angular, utils, etc., and remove eslint globals definition for them. We can require where needed instead.

This looks like a lot of changes since removing the IIFE's gives us back two spaces at the beginning of every line. View the diff [with whitespace changes omitted](https://github.com/DestinyItemManager/DIM/pull/1403/files?w=1) for a better view.

/cc @joshhunt @SunburnedGoose

### CODE REVIEW

File: src/scripts/dimApp.config.js [hunk 1]
```diff
@@ -1,194 +1,170 @@
...
+    };
+
+    var chromeVersion = /Chrome\/(\d+)/.exec($window.navigator.userAgent);
+
+    $rootScope.$on('dim-settings-loaded', function() {
+      var language = dimSettingsService.language;
+      if (chromeVersion && chromeVersion.length === 2 && parseInt(chromeVersion[1], 10) < 51) {
+        dimInfoService.show('old-chrome', {
+          title: $translate.instant('Help.UpgradeChrome'),
+          view: 'views/' + language + '/upgrade-chrome.html?v=$DIM_VERSION',
```
Comment (delphiactual) [inline src/scripts/dimApp.config.js]: require()

File: src/scripts/dimApp.config.js [hunk 2]
```diff
@@ -1,194 +1,170 @@
...
-        $rootScope.$on('$stateChangeSuccess', function() {
-          // if (ga) {
-          //   ga('set', 'page', $location.path());
-            // Disable sending pageviews on state changes for now, over concerns that we'll go over our free GA limits.
-            // ga('send', 'pageview');
-          // }
+      if (dimFeatureFlags.changelogToaster && compare('$DIM_FLAVOR', 'release')) {
+        dimInfoService.show('changelogv$DIM_VERSION'.replace(/\./gi, ''), {
+          title: compare('$DIM_FLAVOR', 'release') ? $translate.instant('Help.Version.Stable') : $translate.instant('Help.Version.Beta'),
+          view: 'views/changelog-toaster' + (compare('$DIM_FLAVOR', 'release') ? '' : '-beta') + '.html?v=v$DIM_VERSION'
```
Comment (delphiactual) [inline src/scripts/dimApp.config.js]: require()

**Comment by kyleshay** [comment]
is there something else we could do instead of writing `const angular = require('angular');` in every single file? or is that just the nature of the beast?

**Comment by SunburnedGoose** [comment]
Every file has to declare its dependencies. What is the ?w=1 thing?

**Comment by delphiactual** [comment]
@SunburnedGoose https://github.com/blog/967-github-secrets

**Comment by bhollis** [comment]
@kyleshay yeah that's just how it is. Each file is a separate CommonJS module, so it has nothing in-scope that it doesn't ask for.

File: src/scripts/dimApp.config.js (hunk 1 above)
Comment (bhollis) [inline src/scripts/dimApp.config.js]: Good idea, but I don't know how webpack can handle a dynamic require.

Comment (delphiactual) [inline src/scripts/dimApp.config.js]: the same way it does in [here](https://github.com/DestinyItemManager/DIM/blob/56970160aa4bea15625a511649aa1ab96c20c32b/src/scripts/shell/dimAppCtrl.controller.js#L137)

**Pull request merged**